library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -1069,10 +1091,7 @@ public void onResult(LocationSettingsResult result) {
                     switch (status.getStatusCode()) {
                         case LocationSettingsStatusCodes.SUCCESS:
                             Log.i(TAG, "All location settings are satisfied.");
-                            // initialize location component wrapper
-                            if (mapboxMap != null) {
-                                mapboxLocationComponentWrapper.init(mapboxMap, getContext());
```

Comment:
Why is this removed here?

Reply:
It does not depend on the location permissions. The decision on whether to return this back is going to be informed from this discussion https://github.com/onaio/kujaku/pull/274#discussion_r263392596

Reply:
OK, moving to https://github.com/onaio/kujaku/pull/274#discussion_r263392596.